Gemfile

```diff
@@ -2,15 +2,19 @@ source 'https://rubygems.org'
 
 gemspec
 gem 'fog-aws'
-gem 'capybara', '~> 2.3.0'
-gem 'database_cleaner', '~> 0.9.1'
-gem 'rspec-rails', '~> 3.2.0'
-gem 'poltergeist'
 gem 'ammeter'
-gem 'test-unit', platforms: [:ruby_22]
 
-group :development do
+group :development, :test do
```

Comment:
I don't know what you mean about the dummy app - it has its own Gemfile (the one you linked) and Gemfile.lock, so it's unaffected by anything going on in the main Slices versions of those files, isn't it? So none of the test-y gems are loaded when that guy runs.